src/pages/home/ReportScreen.js

```diff
@@ -301,17 +301,15 @@ class ReportScreen extends React.Component {
                                     containerHeight={this.state.skeletonViewContainerHeight}
                                 />
                             )}
-                            {this.isReportReadyForDisplay() && !isLoadingInitialReportActions && (
```

Comment:
Can you please summarize whatever question you have and what you have tried so far to solve the problem?

Reply:
Sorry, @0xmiroslav - what is your question? Can you please summarize **what** you are trying to do? Or please ask a single specific question about some changes I've made and we can go from there? I am not able to fully review this PR at the moment and I don't understand at all what we are trying to achieve.

Reply:
Which condition are you referring to? Do you have a good understanding of why it exists or not? Anything I can help clarify?

Reply:
Without looking into whether it is "safe" or not - I'd assume no code is "safe" to remove unless we understand what it does and can explain why it's not needed.

Reply:
Going off of what little has been provided I would say that forcing a re-render with a key could work, but we need to be sure that we are not showing a "composer" that is matched with the previous report. It's possible to transition from one report to another - which is why we hide/show the composer. 

I would probably suggest showing some kind of placeholder composer, but you are welcome to get more feedback in the Slack channel.

@rushatgabhane @Beamanator Was this detail considered during the design/proposal stage?

Reply:
There is some time before the `report.reportID` is correct. Review this and it will hopefully make more sense:

https://github.com/Expensify/App/blob/0422c5fb370762331530573dd32412d2006f80d6/src/pages/home/ReportScreen.js#L154-L162